src/main/java/com/github/jreddit/action/FlairActions.java

```diff
@@ -0,0 +1,674 @@
+package com.github.jreddit.action;
+
+import com.github.jreddit.entity.User;
+import com.github.jreddit.retrieval.ActorDriven;
+import com.github.jreddit.utils.ApiEndpointUtils;
+import com.github.jreddit.utils.restclient.Response;
+import com.github.jreddit.utils.restclient.RestClient;
+
+/**
+ * Flair actions for posts and users
+ *
+ * @author Ryan Delaney (Vitineth)
+ * @since 03/01/2015
```

Comment:
usually the __@since__ tag should be the current version of the release or file in question.